src/main.zig

```diff
@@ -104,21 +104,70 @@ test "CBC mode" {
 
     const z = M.init(key);
 
-    var h = std.crypto.hash.sha2.Sha256.init(.{});
```

Comment:
Why not keep the hash to ensure that we always get the same global output?

Reply:
The initial global output isn't really well-known since it is just the output of the program. I guess it's nice that we can test that it doesn't change over time, but we should really write a second program using a known-good implementation and compare against that output.

I can add it back though.